Library/ViewModels/SearchViewModel.swift

```diff
@@ -187,8 +188,17 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
 
     // KSRAnalytics
 
-    viewWillAppearNotAnimated
-      .observeValues { AppEnvironment.current.ksrAnalytics.trackProjectSearchView() }
+    // Create SignalProducers to emit initial values
+    // for projects and discovery params
+    let projectsProducer = SignalProducer(self.projects).prefix(value: [])
+    let discoveryParamsProducer = SignalProducer(requestFirstPageWith).prefix(value: initialDiscoveryParams)
+
+    discoveryParamsProducer.combineLatest(with: projectsProducer)
```

Comment:
I know this works but I haven't seen this convention used with multiple SignalProducers elsewhere in the app. Can you try to achieve this functionality with signals, perhaps combining them?